libpldmresponder/fru.cpp

```diff
@@ -144,19 +147,28 @@ void FruImpl::buildFRUTable()
         return;
     }
 
+    subscribeFruPresence(inventoryObjPath, fanInterface, itemInterface,
+                         fanHotplugMatch);
+    subscribeFruPresence(inventoryObjPath, psuInterface, itemInterface,
+                         psuHotplugMatch);
+    subscribeFruPresence(inventoryObjPath, pcieAdapterInterface, itemInterface,
+                         pcieHotplugMatch);
+    subscribeFruPresence(inventoryObjPath, panelInterface, itemInterface,
+                         panelHotplugMatch);
+
     fru_parser::DBusLookupInfo dbusInfo;
 
+    // Read the all the inventory D-Bus objects
     try
     {
         dbusInfo = parser.inventoryLookup();
-        objects = pldm::utils::DBusHandler::getInventoryObjects<
-            pldm::utils::DBusHandler>();
+        objects = pldm::utils::DBusHandler::getInventoryObjects();
     }
     catch (const std::exception& e)
     {
-        error(
-            "Failed to build FRU table due to inventory lookup, error - {ERROR}",
-            "ERROR", e);
+        info(
+            "Look up of inventory objects failed and PLDM FRU table creation failed ERROR={ERR_EXCEP}",
```

Comment:
1. format sentence like this => "failed ... creation: {ERROR}"
2. replace e.what() => only 'e'

Reply:
why is this reverted from error to info ?